components/HTTP/src/RESTServer.cpp

```diff
@@ -20,13 +20,14 @@
 #include "FuelGauge.h"
 #include "HttpServer.h"
 #include "Websocket.h"
-#include "WiFi.h"
 #include "defaults.h"
 #include <Storage.h>
 #include <freertos/FreeRTOS.h>
 #include <freertos/task.h>
 #include <iostream>
 
+#include <Network/Network.h>
```

Comment:
```suggestion
#include "Network/Network.h"
```

Reply:
I choose to leave it with `<>` because the headers are outside the source files (`src` is for code, `includes` are headers). What do you think?